### PR TITLE
Update PokerCalculator.java

### PR DESCRIPTION
**Pull request by tcorea**

### CODE REVIEW

File: src/main/java/calculator/PokerCalculator.java [hunk 1]
```diff
@@ -1,18 +1,23 @@
...
+import javax.swing.text.html.HTMLDocument.Iterator;
 
 /**
  * This class will produce the probability based off the
  * users input. Poker Calculator.
  * @author Thomas
  *
  */
-package calculator;
+
```
Comment (vlyandra) [inline src/main/java/calculator/PokerCalculator.java:13]: Did you remove the package declaration for a reason?

File: src/main/java/calculator/PokerCalculator.java [hunk 2]
```diff
@@ -194,22 +494,133 @@ public boolean hasThreeOfKind()
...
+				else if(royalCards.contains(userCards.get(1)))
+						return true;
+				else
+					return false;
+			}
+			else
+				return false;
 	}
 
 	//This will produce the probability of winning.
```
Comment (vlyandra) [inline src/main/java/calculator/PokerCalculator.java:553]: Please use Javadoc comments.

File: src/main/java/calculator/PokerCalculator.java [hunk 3]
```diff
@@ -194,22 +494,133 @@ public boolean hasThreeOfKind()
...
+				royalCards.add(card);//adding the card
+			}
+
+		}
+
+			if(tenCard && jackCard && queenCard && kingCard && aceCard)
+			{
+				if(royalCards.contains(userCards.get(0)))
+						return true;
+				else if(royalCards.contains(userCards.get(1)))
```
Comment (vlyandra) [inline src/main/java/calculator/PokerCalculator.java:544]: Is there a reason this isn't just a logical OR?

**Review by vlyandra — CHANGES_REQUESTED** [review]
This code needs to be linted and fixed to adhere to the Google Java Style guide: https://google.github.io/styleguide/javaguide.html

Something I can see immediately is that the brace style is generally incorrect: https://google.github.io/styleguide/javaguide.html#s4.1-braces

I believe we also discussed using the two plus two strategy for probability calculation, using a lookup table, instead of the naive solution?

File: src/main/java/calculator/PokerCalculator.java [hunk 4]
```diff
@@ -105,32 +131,86 @@ public boolean hasHighCard()
 	 */
 	public boolean hasPair()
```
Comment (vlyandra) [inline src/main/java/calculator/PokerCalculator.java:132]: All of these hand evaluation functions should have tests.

File: src/main/java/calculator/PokerCalculator.java (hunk 3 above)
Comment (tcorea) [reply src/main/java/calculator/PokerCalculator.java:544]: i guess we can use or statement. I have some junit tests for this method and will test it out just to be sure. lol

File: src/main/java/calculator/PokerCalculator.java (hunk 2 above)
Comment (tcorea) [reply src/main/java/calculator/PokerCalculator.java:553]: ok i will make the corrections tonight

File: src/main/java/calculator/PokerCalculator.java (hunk 1 above)
Comment (tcorea) [reply src/main/java/calculator/PokerCalculator.java:13]: no i dont think so maybe a mistake mistyped or something

File: src/main/java/calculator/PokerCalculator.java (hunk 4 above)
Comment (tcorea) [reply src/main/java/calculator/PokerCalculator.java:132]: Yes I have tests for all of them. i've tested them before commiting and the pr. Do you need the junit test cases ?

Comment (vlyandra) [reply src/main/java/calculator/PokerCalculator.java:132]: Tests should always be committed with the code.

**Comment by tcorea** [comment]
> This code needs to be linted and fixed to adhere to the Google Java Style guide: https://google.github.io/styleguide/javaguide.html
> 
> Something I can see immediately is that the brace style is generally incorrect: https://google.github.io/styleguide/javaguide.html#s4.1-braces
> 
> I believe we also discussed using the two plus two strategy for probability calculation, using a lookup table, instead of the naive solution?

yes i was gonna continue to come up with a ratio which in turn gets us a percentage, by finding the desirabe outcomes/unwanted outcomes. . I trying looking for a two plus two strategy but have a hard time coming up with a solution of sorts do you any good references for that strategy ?

**Pull request closed without merging**